working-days/working-days.component.ts

```diff
@@ -0,0 +1,37 @@
+import { Component, OnInit } from '@angular/core';
+import { MatCheckboxModule } from '@angular/material/checkbox';
+import { MatSelectModule } from '@angular/material/select';
+import { MatButtonModule } from '@angular/material/button';
+import { MatFormFieldModule } from '@angular/material/form-field';
+import { MatTableModule } from '@angular/material/table';
+import { Router, ActivatedRoute } from '@angular/router';
+import { NgModule } from '@angular/core';
+
+@NgModule({
```

Comment:
This is a component so you should not be adding @NgModule to this. Please again see other components for reference.